src/term.js

```diff
@@ -732,75 +732,83 @@ Terminal.prototype.open = function(parent) {
   // Draw the screen.
   this.refresh(0, this.rows - 1);
 
-  // Initialize global actions that
-  // need to be taken on the document.
-  this.initGlobal();
+  if (this.options.noEvents) {
```

Comment:
Shouldn't this be negative - `if (!this.options.noEvents)`?

Also, maybe the options should be positively named, so you specify something like `events=false` instead of `noEvents=true`.